### ISSUE TITLE
Worse Lighthouse score due to image placeholders not served in next-gen format

### ISSUE THREAD

**Issue by mcmxcdev** [xiphux/svimg#22]
![WhatsApp Image 2022-07-06 at 6 42 38 PM](https://user-images.githubusercontent.com/16797721/177601505-4fa7f54d-5dad-49f2-8ca4-15eb13e9e1fa.jpeg)

**Comment by xiphux** [comment]
Yeah - I'd like to do this, but this is a little less straightforward than the srcset for the main image since the image is embedded into the document, so embedding multiple formats would be transferring a bunch of unused content to the browser.

Switching over to WebP period would mean dropping IE11 support (which I don't feel *that* bad about), but would also be problematic for Safari on OSX earlier than 11.

**Comment by mcmxcdev** [comment]
Thanks for letting me know about the reasoning!

I think requiring macOS 11 Big Sur or later shouldn't be an issue. As a fact, I use a Big Sur as my secondary device, and Apple already disallows my device to upgrade to any newer macOS versions. So there is a certain push for newer hardware already anyways.

**Comment by ohbob** [comment]
> You can serve the 10px placeholder from jpg and that's that. As the rest of images in the image tag will go trough the srcset check and will drop unsupported images.

**Comment by xiphux** [comment]
I don't think serving the placeholder as jpg would resolve the Lighthouse warning though? The message says that WebP/AVIF are nextgen and PNG/JPEG are not.
